pidng/core.py

```diff
@@ -250,182 +331,110 @@ def __process__(self, input_file, processing):
-
-            ccm2 = [[5603, 10000], [-1351, 10000], [-600, 10000],
-                    [-2872, 10000], [11180, 10000], [2132, 10000],
-                    [600, 10000], [453, 10000], [5821, 10000]]
-
-            fm = True
-
-            fm1 = [[7889, 10000], [1273, 10000], [482, 10000],
-                   [2401, 10000], [9705, 10000], [-2106, 10000],
-                   [-26, 10000], [-4406, 10000], [12683, 10000]]
-
-            fm2 = [[6591, 10000], [3034, 10000], [18, 10000],
-                   [1991, 10000], [10585, 10000], [-2575, 10000],
-                   [-493, 10000], [-919, 10000], [9663, 10000]]
-
-            ci1 = 17
-            ci2 = 21
-
-        else:
-            as_shot_neutral = [[10043, 10000], [16090, 10000], [10000, 10000]]
-
-            ccm1 = [[19549, 10000], [-7877, 10000], [-2582, 10000],
-                    [-5724, 10000], [10121, 10000], [1917, 10000],
-                    [-1267, 10000], [-110, 10000], [6621, 10000]]
-
-            ccm2 = [[13244, 10000], [-5501, 10000], [-1248, 10000],
-                    [-1508, 10000], [9858, 10000], [1935, 10000],
-                    [-270, 10000], [-1083, 10000], [4366, 10000]]
-            ci1 = 1
-            ci2 = 23
-
-        baseline_exp = 1
-        
-        camera_calibration = [[1, 1], [0, 1], [0, 1],
-                              [0, 1], [1, 1], [0, 1],
-                              [0, 1], [0, 1], [1, 1]]
-
-        if self.maker_note:
-            gain_r = int(float(self.maker_note['gain_r'])*1000)
-            gain_b = int(float(self.maker_note['gain_b'])*1000)
-
-            baseline_exp = int(self.maker_note['ev'])
-
-            as_shot_neutral = [[1000, gain_r], [1000, 1000], [1000, gain_b]]
-
-        compression_scheme = 7 if compress else 1
-
         if compress:
             from ljpegCompress import pack16tolj
-            tile = pack16tolj(rawFrame, int(width*2),
-                              int(length/2), bpp, 0, 0, 0, "", 6)
+            tile = pack16tolj(raw_frame, int(width*2),
+                              int(height / 2), bpp, 0, 0, 0, "", 6)
         else:
             if (bpp - sensor_bpp) >= 0:
-                rawFrame = rawFrame << (bpp - sensor_bpp)
+                raw_frame = raw_frame << (bpp - sensor_bpp)
             else:
-                rawFrame = rawFrame >> abs(bpp - sensor_bpp)
+                raw_frame = raw_frame >> abs(bpp - sensor_bpp)
 
             if bpp == 8:
-                tile = (rawFrame//255).astype('uint8').tobytes()
+                tile = (raw_frame//255).astype('uint8').tobytes()
             elif bpp == 10:
-                tile = pack10(rawFrame).tobytes()
+                tile = pack10(raw_frame).tobytes()
             elif bpp == 12:
-                tile = pack12(rawFrame).tobytes()
+                tile = pack12(raw_frame).tobytes()
             elif bpp == 14:
-                tile = pack14(rawFrame).tobytes()
+                tile = pack14(raw_frame).tobytes()
```

Comment:
This may be outside the scope of this PR, but `pack14` returns None, not an array, so this will explode.

Reply:
@itsthejoker
All your points made are correct.
I do no longer use PiDNG and have no funding.
Are you interested in taking over?

Cheers,
Volker

Reply:
I... suppose? Do you have the ability to give me merge permissions? If so, then I can finish the PR tomorrow and start hacking to do that refactor.

Reply:
@schoolpost 
Do you have any objections?

Reply:
If not please grant @itsthejoker the perimissions he need.

Cheers,
Volker

Reply:
Hi Everyone, 

@volkerjaenisch safe to say you would like to be removed from the project going forward in this case?

@itsthejoker I will grant you the necessary permissions, would like to have a discussion before this however.

Reply:
@schoolpost sent you an email to the address listed in Git.